purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -348,7 +348,7 @@ internal fun Purchases.logInWith(
  * @param [onError] Will be called after the call has completed with an error.
  */
 @Suppress("unused")
-internal fun Purchases.logOutWith(
+fun Purchases.logOutWith(
```

Comment:
just curious, does this return the newly-created anonymous user's purchaser info?

Reply:
it deletes all data for the current user, then generates a new anonymous id, and returns the purchaserinfo for that new anonymous id